frontend/src/components/pages/profile/components/friendsList/FriendsList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useMemo } from 'react';
+import Box from '@material-ui/core/Box';
+import { Link } from 'react-router-dom';
+import { UserPhoto } from '../../../../shared/userPhoto/UserPhoto';
+import './friendsList.scss';
+import { ListOfFriends } from '../../../../interfaces/Interface';
+
+const FriendsList: React.FC<ListOfFriends> = ({ friends }) => {
+  const sortedFriends = useMemo(() => {
```

Comment:
I am not sure that current logic should be stored here, in that component. @oleksandrstarov what do you think?

Reply:
I would say it depends on usage
If this component will handle click with presenting all friends list - it can be kept here, otherwise - moved to the service that will fetch data and converted like this on response handling